docs(auth): note that resetPasswordForEmail doesn't send email for un… (#48800)
supabase/supabase: apps/docs/content/guides/auth/passwords.mdx

### Resetting a password

<Admonition type="note">

> To prevent user enumeration, `resetPasswordForEmail()` doesn't reveal whether an account exists for the given email address. If no user is associated with the address, Supabase Auth doesn't send an email, but the method still returns without an error.

Reviewer: ```suggestion
To prevent user enumeration, `resetPasswordForEmail()` doesn't reveal whether an account exists for the given email address.

When no user is associated with the address, Supabase Auth won't send an email, though the method will still return without an error.
```
